src/components/ThemeSwitch/ThemeSwitch.js

```diff
@@ -0,0 +1,47 @@
+import React from 'react';
+import { useTranslation } from 'react-i18next';
+
+import { Option } from '@ui5/webcomponents-react/lib/Option';
+import { Select } from '@ui5/webcomponents-react/lib/Select';
+import { setTheme } from '@ui5/webcomponents-base/dist/config/Theme.js';
+import InformationDialog from '../InformationDialog/InformationDialog';
+
+const ThemeSwitch = ({ dialogRef, storedTheme = localStorage.getItem('reactSeedSelectedTheme') }) => {
+  const { t } = useTranslation();
+  setTheme(storedTheme ? storedTheme : 'sap_fiori_3');
```

Comment:
Can you change this to `useEffect`? This way `setTheme` is only called on first load and when `storedTheme` really changed.

Reply:
I updated it, is it like you suggested?